package.json

```diff
@@ -15,9 +15,11 @@
   "scripts": {
     "postinstall": "npm ls --depth=0",
     "build": "node tasks/build.js",
-    "start": "node tasks/serve.js",
+    "start": "node tasks/start.js",
+    "serve": "node tasks/serve.js",
+    "pretest": "npm run build",
     "test": "npm run lint && jest",
-    "lint": "npm run lint:js && npm run lint:scss && npm run lint:html",
+    "lint": "npm run lint:js && npm run lint:scss && npm run lint:html --ignore-scripts",
```

Comment:
When running the "All in one" task `npm test` we have already built the site using `pretest`

So I've added `--ignore-scripts` to skip `prelint:html` to avoid a double build

Reply:
**question** If we're doing this to avoid the double build when running `test`, because both `pretest` and `prelint:html` run a build, would this addition be better suited to the `test` command? Can it run `npm run  lint -- --ignore-scripts` to append `ignore-scripts` to the `lint` command? No worries if it can't though.

Reply:
Does it work? Happy to switch it

Only worry is `"lint"` being reordered in future and the flag goes to the wrong script

Reply:
Seems to work, yup (🤯), but good shout on the reordering. Feels odd that, when run through `lint`, `lint:html` won't build, but when run on its own it will. If it becomes an issue we can always adjust then.

Reply:
Yeah maybe aligning it with `govuk-frontend` was the wrong move?

Where only `"test"` runs a build via `"pretest"`

Reply:
I'm not sure I understand your last answer, sorry 😔

Reply:
Ah sorry. I mean the linting tasks on `govuk-frontend` don't run a build, so I copied that here

Might be worth trying to tighten up what runs when